datadog.go

```diff
@@ -69,6 +69,10 @@ type Options struct {
 
 	// Tags specifies a set of global tags to attach to each metric.
 	Tags []string
+
+	// GlobalTags holds a set of tags that will be automatically applied to
+	// all spans.
```

Comment:
Let's make sure it's clear that these tags don't automatically get applied to (OpenCensus) spans, but only to their resulting Datadog-exported counter-spans:
```go
// GlobalTags holds a set of tags that will automatically be applied to all
// exported spans.
```